<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Third_Dialog</class>
 <widget class="QDialog" name="Third_Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>533</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QGroupBox" name="EnterInfo_1">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>40</y>
     <width>421</width>
     <height>251</height>
    </rect>
   </property>
   <property name="title">
    <string>GroupBox</string>
   </property>
   <widget class="QLabel" name="Enter_Passwor_1">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>130</y>
      <width>121</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Enter Password:</string>
    </property>
   </widget>
   <widget class="QLabel" name="Enter_ID_2">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>50</y>
      <width>120</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Enter Name:</string>
    </property>
   </widget>
   <widget class="QLabel" name="Enter_Name_1">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>90</y>
      <width>121</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Enter EmploID:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_NAME">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>50</y>
      <width>113</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_2_ID">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>90</y>
      <width>113</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_3_PASSWORD">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>130</y>
      <width>113</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="Save_Button">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>170</y>
      <width>113</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>Save!</string>
    </property>
   </widget>
   <widget class="QLabel" name="Status_Label">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>200</y>
      <width>151</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>SQL Status:</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
